R/read_habitatdata.R

```diff
@@ -40,6 +40,17 @@
 #'   each of them. The variable \code{certain} will be \code{FALSE} if
 #'   the original code consists of 2 or 3 possible vegetation types, and \code{TRUE}
 #'   if only one vegetation type is provided.
+#'   \itemize{
```

Comment:
I think this deserves an item on its own (no subitem of the uncertain case): AFAIK the codes with a comma are only used in the generate.Rmd, the fields `hm_stdized$habitatmap_polygons$description_orig` and `hm_stdized$habitatmap_types$code_orig` still mention the original version with an underscore. 

So maybe: 
```
#'     \item Some polygons contain both a standing water habitat type 
#'     and \code{rbbmr}, e.g.:
#'     \code{3130_rbbmr},
#'     \code{3140_rbbmr},
#'     \code{3150_rbbmr} and
#'     \code{3160_rbbmr}.
#'     Since \code{habitatmap_stdized_2020_v1}, the two vegetation
#'     types \code{31xx} and \code{rbbmr} are split up and one row is 
#'     created for each of them. 
#'     The variable \code{certain} in this case will be \code{TRUE} for both types.
```